pkg/scalers/cron_scaler_test.go

```diff
@@ -31,6 +31,8 @@ var testCronMetadata = []parseCronMetadataTestData{
 	{validCronMetadata, false},
 	{map[string]string{"timezone": "Asia/Kolkata", "start": "30 * * * *", "end": "45 * * * *"}, true},
 	{map[string]string{"start": "30 * * * *", "end": "45 * * * *", "desiredReplicas": "10"}, true},
+	{map[string]string{"timezone": "Asia/Kolkata","start": "30-33 * * * *", "end": "45 * * * *", "desiredReplicas": "10"}, true},
+	{map[string]string{"timezone": "Asia/Kolkata","start": "30 * * * *", "end": "45-50 * * * *", "desiredReplicas": "10"}, true},
```

Comment:
Could you please add some incorrect entries, so we can be sure that the validation is working correctly? Thanks!

Reply:
Hi @zroubalik , the added entries are incorrect as they have range values. Could you please elaborate what other incorrect entries do you expect? Are these good: 

> {map[string]string{"timezone": "Asia/Kolkata","start": "-33 * * * *", "end": "45 * * * *", "desiredReplicas": "10"}

> {map[string]string{"timezone": "Asia/Kolkata","start": "30 * * * *", "end": " - * * * *", "desiredReplicas": "10"}

> {map[string]string{"timezone": "Asia/Kolkata","start": "33 * * * *", "end": "45 * * -3 *", "desiredReplicas": "10"}

Reply:
Yeah, these 3 are good!